content/en-us/projects/managing.md

```diff
@@ -179,7 +177,7 @@ Modeling](/art/modeling).
 
 <img src="../assets/misc/Building-Best-Practices-Example-Parts.jpg" />
 
-### Default Materials
+### Applying Materials
```

Comment:
This is more about material usage in general, including custom materials, not just the defaults.